manuscript.tex

```diff
@@ -60,16 +60,8 @@ \chapter{Chapter 2: Collaborating on code and data}
 %	CHAPTER 3
 %----------------------------------------------------------------------------------------
 
-\chapter{Chapter 3: Evaluating impact through research design}
-\label{ch:3}
 
-\input{chapters/research-design.tex}
-
-%----------------------------------------------------------------------------------------
-%	CHAPTER 4
-%----------------------------------------------------------------------------------------
-
-\chapter{Chapter 4: Sampling, randomization, and power}
+\chapter{Chapter 3: Establish a measurement framework}
```

Comment:
Currently, the chapter title is **Establishing a measurement framework**. I do not love that title at all. It is not informative to anyone who had not already read the chapter. What about:
* **Planning project data**
* **Planning a data ecosystem**
* **Planning a data work**
* **Understanding and implementing data requirements**

Reply:
I actually like establishing a measurement framework, that is what this is about (establishing what data sources are needed, linking them together, thinking through units of analysis, drawing samples, assigning tmt etc). Chapter 2 is 'setting the stage for successful data work'; planning data work seems too similar.

Reply:
Hopefully once @bbdaniels makes edits tying the data planning more directly into the analytical framework this chapter title will seem a better fit!

Reply:
ok, I am happy with the new title `Establishing a measurement framework` as everyone else seems to prefer it.